.grit/test/test.yaml

```diff
@@ -0,0 +1,82 @@
+# this file is used for integration testing of `grit patterns test --watch`
```

Comment:
Please don't invent a new way to do integration tests. We already have fixtures and existing conventions. Look at them and follow that.